stable/mlrun-kit/templates/kubeflow/configmaps/pipeline-install-config.yaml

```diff
@@ -0,0 +1,34 @@
+{{- if .Values.pipelines.enabled -}}
+apiVersion: v1
+#eliyahu
+data:
+  minioServiceHost: "minio.mlrun.svc.cluster.local"
+  minioServicePort: "9000"
+  ConMaxLifeTime: 120s
+  appName: pipeline
+  appVersion: 1.8.3
+  autoUpdatePipelineDefaultVersion: "true"
+  bucketName: mlrun
```

Comment:
have these as values in the values file. And what exactly is `autoUpdatePipelineDefaultVersion`? might want to have that false